src/web/components/ArticleHeadline.stories.tsx

```diff
@@ -46,40 +46,6 @@ export const ArticleStory = () => (
 );
 ArticleStory.story = { name: 'Article' };
 
-export const oldHeadline = () => (
```

Comment:
This story seemed a little unnecessary since the age warning is added outside this component so I removed it but let me know if that's a bad idea